blog/2017-12-07-streaming-to-elasticsearch.adoc

```diff
@@ -0,0 +1,248 @@
+
+&nbsp; +
+
+.A Simplified topology
+[#img-general]
+[caption="Figure 2: "]
+image::dbz-to-multiple-simplified.svg[Scenario topology]
+
+== Configuration
+We will use this https://github.com/debezium/debezium-examples/tree/master/unwrap-smt[compose] for a fast deployment of the demo.
+The deployment consists of following Docker images:
+
+* https://hub.docker.com/r/debezium/zookeeper/[Apache ZooKeeper]
+* https://hub.docker.com/r/debezium/kafka/[Apache Kafka]
+* An https://github.com/debezium/debezium-examples/tree/master/unwrap-smt/debezium-jdbc[enriched] Kafka Connect / Debezium https://hub.docker.com/r/debezium/connect/[image] with changes
+** PostgreSQL JDBC driver placed into `/kafka/libs` directory
+** https://docs.confluent.io/current/connect/connect-jdbc/docs/index.html[Kafka Connect JDBC Connector] (developed by https://www.confluent.io/[Confluent]) placed into `/kafka/connect/kafka-connect-jdbc` directory
+* Pre-populated MySQL used in our link:docs/tutorial[tutorial]
+* Empty PostgreSQL
+* Empty ElasticSearch
+
+We again use the `UnwrapFromEnvelope` transformation to automatically extract a new row record and thus effectively convert the Debezium complex record into a simple one consumable by other connectors. Both JDBC and Elasticsearch connectors expect the same format.
+
+== Example
+Let's move directly to our exmaple as that's where the changes are visible.
+
+First of all we need to deploy all components.
+[source,bash,indent=0]
+----
+export DEBEZIUM_VERSION=0.6
+docker-compose up
+----
+
+When all components are started we are going to register the Elasticsearch Sink connector writing into the ElasticSearch instance.
+We want to use the same key (primary id) in the source and both PostgreSQL and Elasticsearch.
+[source,bash,indent=0]
+----
+curl -i -X POST -H "Accept:application/json" -H  "Content-Type:application/json" http://localhost:8083/connectors/ -d @es-sink.json
+----
+
+Using this registration request:
+
+[source,json,indent=0]
+----
+{
+    {
+        "name": "elastic-sink",
+        "config": {
+            "connector.class": "io.confluent.connect.elasticsearch.ElasticsearchSinkConnector",
+            "tasks.max": "1",
+            "topics": "customers",
+            "connection.url": "http://elastic:9200",
+            "transforms": "unwrap,key",
+            "transforms.unwrap.type": "io.debezium.transforms.UnwrapFromEnvelope",           (1)
+            "transforms.key.type": "org.apache.kafka.connect.transforms.ExtractField$Key",   (2)
+            "transforms.key.field": "id",                                                    (2)
+            "key.ignore": "false",                                                           (3)
+            "type.name": "kafka-connect"                                                     (4)
+        }
+    }
+}
+----
+
+The request configures these options:
+
+1. unwrapping Debezium's complex format into a simple one
```

Comment:
How about "Extracting only the new rows state from Debezium's change data message"